### PR TITLE
feat: Implement news page template - EXO-65437 - Meeds-io/MIPs#68

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ License along with this software; if not, write to the Free
+ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_objects_1_8 http://www.exoplatform.org/xml/ns/gatein_objects_1_8"
+      xmlns="http://www.exoplatform.org/xml/ns/gatein_objects_1_8">
+    <name></name>
+    <container
```
Comment (azayati) [inline webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]: indentation

File: webapp/src/main/webapp/skin/less/newsDetails.less [hunk 2]
```diff
@@ -600,4 +600,34 @@
 
 .news-details-toolbar {
   position: inherit!important;
+}
+
+#topNewsPage {
```
Comment (azayati) [inline webapp/src/main/webapp/skin/less/newsDetails.less]: are these css classes related to news detail ?

**Review by azayati — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/skin/less/newsDetails.less (hunk 2 above)
Comment (GouadriaHanen) [reply webapp/src/main/webapp/skin/less/newsDetails.less]: it is related to new page template

**Review by azayati — APPROVED** [review]
LGTM
